test/e2e/specs/specs-wpcom/wp-calypso-gutenberg-editor-tracking-spec.js

```diff
@@ -23,6 +23,8 @@ const host = dataHelper.getJetpackHost();
 // will be deployed to Dotcom) as well as the current version of Gutenberg.
 const gutenbergUser =
 	process.env.GUTENBERG_EDGE === 'true' ? 'gutenbergSimpleSiteEdgeUser' : 'gutenbergSimpleSiteUser';
+const siteEditorUser =
+	process.env.GUTENBERG_EDGE === 'true' ? 'siteEditorSimpleSiteUser' : 'siteEditorSimpleSiteUser';
```

Comment:
Should we create a Gutenberg Edge user? Back then in https://github.com/Automattic/wp-calypso/pull/48365 I've only created a non-edge user.

Reply:
If we want to run the `Tracking Site Editor` suite as a part of upgrade testing then yeah, we'd need an edge user as well. If not, we should probably skip it:
```js
describe( 'Tracking Site Editor: @parallel', function () {
	before( async function () {
		if ( process.env.GUTENBERG_EDGE === 'true' ) {
			this.skip();
		}
	} );

	// ...
} );
```